### ISSUE TITLE
paginate="true" breaks collections tag

### ISSUE THREAD

**Issue by pmeissner** [statamic/v2-hub#796]
### Steps to reproduce
1. Create a collections tag:
```{{ collection:blog paginate="true" }}
{{ title }}
{{ /collection:blog }}```

### Expected behaviour
Return all entries from the blog.

### Actual behaviour
Only pulls the blog index page's variables.

### Server configuration
MAMP PRO - PHP 7.0

**Statamic version:**
2.0.11

**Updated from an older Statamic or fresh install:**
Update from old

NOTE: This works
```{{ collection:blog }}
{{ title }}
{{ /collection:blog }}```

**Comment by jackmcdade** [comment]
When you paginate, it pushes everything into Entries. Check the docs: <https://docs.statamic.com/reference/tags/collection#pagination>
```
{{ collection:blog paginate="true" }}
  {{ entries }}
    {{ title }}
  {{ /entries }}
{{ /collection:blog }}
```

**Comment by pmeissner** [comment]
Then I'm back to my scope problem. How do I keep the variables within those "entries" tags from grabbing the pages variables?

**Comment by jackmcdade** [comment]
Right. That's a separate issue. I just want to be clear that the bug isn't with "default" critical behavior, which would mean an emergency release. Back to testing and Slack on the scoping issue behavior...

**Comment by jasonvarga** [comment]
The scoping issue appears to be fixed for 2.1.7
